src/Mod/Spreadsheet/Gui/SheetModel.cpp

```diff
@@ -78,6 +78,22 @@ int SheetModel::columnCount(const QModelIndex &parent) const
 }
 
 
+QVariant format_cell_display(QString value, const Cell * cell)
+{
+    std::string alias;
+    Base::Reference<ParameterGrp> hGrpSpreadsheet = 
+        App::GetApplication().GetUserParameter().GetGroup("BaseApp")->GetGroup("Preferences")->GetGroup("Mod/Spreadsheet");
+    if (cell->getAlias(alias) && hGrpSpreadsheet->GetBool("showAliasName", false)) {
+        QString formatStr = QString::fromStdString(hGrpSpreadsheet->GetASCII("DisplayAliasFormatString", "%V = %A"));
+        if (formatStr.contains(QLatin1String("%V")) || formatStr.contains(QLatin1String("%A"))) {
+            formatStr.replace(QLatin1String("%A"), QString::fromStdString(alias));
+            formatStr.replace(QLatin1String("%V"), value);
```

Comment:
Considering this function will be called very very often, could be good optimizing it regarding runtime execution. Lot of things could be cached.

Reply:
I'm not sure to get what could be cached... the `QLatin1String` call ? it doesn't seems that expensive to me and done only twice...
or is it more global changing from one call of this function to another ?

Reply:
At least `hGrpSpreadsheet` could be made `static`.

Also I wonder now : when user switches `showAliasName` (from true to false or oppositely) when a spreadsheet is open, do the cells correctly update immediately ?

Reply:
static look fine

we need to, in the preference windows, use OK (which close the windows) to update the formatting, only toggle it don't work, nor using apply. I do think that toggling it should not change it (even if that would be cool, but then the "apply" button, in this case, would have no purpose). However the fact that the "apply" button don't actually apply it is not good, but I've no idea what to fix to make it work.